### PR TITLE
Change deployment configurations

### PR DESCRIPTION
**Pull request by ezhivitsa**
@umitskevich, @anorsich  Please, review

### CODE REVIEW

File: deploy/app/deploy-nginx.yml [hunk 1]
```diff
@@ -2,20 +2,44 @@
...
+    - { role: nginx, when: run_roles == "y" }
   tasks:
+    - name: Assures /etc/nginx/ssl exists
+      file: path=/etc/nginx/ssl state=directory
+      when: server_setup_ssl
+
+    - name: Copy ssl certificate to /etc/nginx/ssl
+      copy: src=ssl-keys/{{item.src}} dest={{item.dest}} owner=root mode=744
+      with_items:
+        - {src: 'app.crt', dest: '{{ssl_cert_path}}'}
```
Comment (anorsich) [inline deploy/app/deploy-nginx.yml]: We copy `app.crt`, but in the nginx config below we use:
- another path (it includes domain of specific app)
- another name

Did you test it? I like an idea to autogenerate lets encrypt certificate (we does it on one of our new products, ask @KuhArt for more details). Also, aside from this: let's encrypt now supports wildcard certificates, so it is not necessary to generate one certificate per domain, we can generate wildcard.
For let's encrypt we might need to setup a separate repo, which will have: 
- guide how to create a certificate 
- some kind of ansible automation to regenerate certificate on server as it expires 

I remember @gavruk wrote an article about this on our internal blog.

File: deploy/app/bin/deploy-staging.sh [hunk 2]
```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-ansible-playbook ./deploy-app.yml -i ./hosts/staging -u root --extra-vars "env=staging" "$@"
+ansible-playbook ./deploy-app.yml -i ./hosts/staging --extra-vars "env=staging" "$@"
```
Comment (anorsich) [inline deploy/app/bin/deploy-staging.sh]: we might need to use `./deploy-app.sh` here

Comment (ezhivitsa) [reply deploy/app/bin/deploy-staging.sh]: @anorsich How do you propose to use `deploy-app.sh`? Do you mean, that we can delete bash file `deploy-staging.sh`?

File: deploy/app/vars/main.yml [hunk 3]
```diff
@@ -14,5 +15,9 @@ web_ports: [ 6100 ]
...
 landing_ports: [ 6200 ]
 
-ssl_key_path: "/etc/letsencrypt/live"
-ssl_cert_path: "/etc/letsencrypt/live"
+ssl_key_path: "/etc/letsencrypt/live/paralect.com"
+ssl_key: "fullchain.pem"
+ssl_cert_path: "/etc/letsencrypt/live/paralect.com"
+ssl_cert: "privkey.pem"
+
+letsencrypt_email: "ship@test.com"
```
Comment (anorsich) [inline deploy/app/vars/main.yml]: Let's change it to andrew@paralect.com

**Pull request merged**